ichnaea/data/tests/test_public.py

```diff
@@ -266,8 +266,8 @@ def test_modified_station(self, session, redis_client, cellarea_queue):
         assert wcdma.lon == 23.4123167
         assert wcdma.radius == 0
         assert wcdma.samples == 6
-        assert wcdma.created == datetime(2019, 9, 11, 16, 49, 24, tzinfo=UTC)
-        assert wcdma.modified == datetime(2019, 10, 3, 16, 31, 56, tzinfo=UTC)
+        assert wcdma.created == datetime(2019, 9, 11, 16, 49, 24, tzinfo=ZoneInfo("UTC"))
+        assert wcdma.modified == datetime(2019, 10, 3, 16, 31, 56, tzinfo=ZoneInfo("UTC"))
```

Comment:
These are also too long now.

An alternate would be to add a global for ``UTC`` near the top of the file:

```
UTC = ZoneInfo("UTC")
```

Then the lines would go back to the shorter versions.

Reply:
Thanks for the support :smiley: